fastlane/lib/fastlane/runner.rb

```diff
@@ -258,12 +258,11 @@ def execute_flow_block(block, current_platform, lane, parameters)
 
     def verify_supported_os(name, class_ref)
       if class_ref.respond_to?(:is_supported?)
-        if Actions.lane_context[Actions::SharedValues::PLATFORM_NAME]
```

Comment:
I understand that instead of raising an exception we show a warning. But why are the other changes necessary here?